### PR TITLE
Shows possible answers inline

### PR DESCRIPTION
**Pull request by filipeximenes**

### CODE REVIEW

File: fixmydjango/__init__.py [hunk 1]
```diff
@@ -23,29 +23,29 @@
...
+    {% for error in errors %}
+        <h3>Django {{ error.django_version }}</h3>
+        {% for answer in error.answers %}
+            <b>Answer {{ forloop.counter }}</b>
+            <pre>{{ answer.message }}</pre>
+        {% endfor %}
+    {% endfor %}
+
+    <h3 style="display: block;">
+        If none of this works, <a style="margin: 0; padding: 0;" href="{{ admin_url }}">click here</a> to request help.
```
Comment (fjsj) [inline fixmydjango/__init__.py]: How `admin_url` leads to help? Previously `admin_url` was to add errors directly on `/admin`.

Comment (filipeximenes) [inline fixmydjango/__init__.py]: This leads to the form page where you can submit the error

Comment (fjsj) [inline fixmydjango/__init__.py]: so maybe it's better to change that variable name, that used to mean something else

Comment (filipeximenes) [inline fixmydjango/__init__.py]: Fixed

**Comment by fjsj** [comment]
Thanks. Merged.

**Pull request merged**